content/en/reference/built-in-actors/protocol-api/index.md

```diff
@@ -1329,3 +1329,4 @@ Results:
   - `struct` BatchReturn
     - `uint32` SuccessCount - total successes in the batch
     - `struct` FailCodes[] {`uint32` idx, `uint32` code} -  list of failure code and index for all failures in batch.
+<!--REVIEWED!-->
```

Comment:
Skipped pre-commit lint on this review for two reasons:

1. Spell check flagged literally every single function term (300+) as wrong. We could add all of these two the dictionary, or we could reformat each codeing term like `sampleFunctionName`. Either way, this is a limit of the spell checker
1. There are two level 3 (###) headers for `WithdrawBalance` on this page, which was flagged by the MD format checker. One is the miner actor method, the other is the storage market actor method. Either way, we can't have duplicated headers on the same page, cus the link checker doesn't like it since that can lead to links that refer to those headers not working, etc.. So, we should figure out what to do about this.